tests/features/browserstack-local.js

```diff
@@ -0,0 +1,57 @@
+/*eslint-env node */
+var Cucumber = require('cucumber');
+var assert = require('cucumber-assert');
+var webdriver = require('selenium-webdriver');
+
+Cucumber.defineSupportCode(function({setDefaultTimeout}) {

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->